### ISSUE TITLE
Onboarding: add steps/progress

### ISSUE THREAD

**Issue by NielLiesmons** [nostrdesignguide/nostrdesignguide.github.io#11]
Add either the steps to go through or a progress bar to the reference designs in the "Onboarding" tab.  
I can also do it if needed.

**Comment by weex** [comment]
This would be great to see. Onboarding is still a big pain point.

**Comment by holokat** [comment]
Yeah we can definitely include this. I think there are all sorts of stepper designs that we can add to reference designs. If you haven't already, I will take a crack at it, otherwise feel free to do a PR 🙏
